code/game/objects/items/miscellaneous.dm

```diff
@@ -47,10 +47,10 @@
 /obj/item/choice_beacon/proc/spawn_option(obj/choice,mob/living/M)
 	var/obj/new_item = new choice()
 	var/obj/structure/closet/supplypod/bluespacepod/pod = new()
-	pod.explosionSize = list(0,0,0,2)
+	pod.explosionSize = list(0,0,0,0)
 	new_item.forceMove(pod)
-	var/msg = "<span class = danger>After making your selection, you notice a strange target on the ground. It might be best to step back!</span>"
-	if (ishuman(M))
+	var/msg = "<span class=danger>After making your selection, you notice a strange target on the ground. It might be best to step back!</span>"
```

Comment:
does this need to be wrapped in `'danger'` to work?

Reply:
there's also a userdanger idk the difference

Reply:
userdanger is bigger and more bold.